toolkit/templates/proposition-header.html

```diff
@@ -9,26 +9,32 @@
         <li>
           <a href="{{ url_for('external.help') }}">Help</a>
         </li>
-        {% if (current_user.is_authenticated() if logged_in is not defined else logged_in) %} {% if (role or current_user.role) == 'buyer' %}
```

Comment:
If I understand this line correctly, the is_authenticated flag is only used if logged_in is not already present, so instead of changing the pages builder template, could we instead just make the new assignment line also use logged_in if it's defined?

Reply:
I thought of an even better way, using `default` http://jinja.pocoo.org/docs/2.10/templates/#default

Reply:
Oooh